server/backend/ios_notifications/ios_apns/iosAPNsService.go

```diff
@@ -27,23 +26,17 @@ type Service struct {
 // and the request ID.
 func (s *Service) RequestGradeUpdateForDevice(deviceID string) error {
 	campusRequestToken, err := s.Repository.CreateCampusTokenRequest(deviceID)
-
 	if err != nil {
 		log.WithError(err).Error("Could not create campus token request")
 		return ErrCouldNotCreateTokenRequest
 	}
 
 	notification := model.NewIOSNotificationPayload(deviceID).Background(campusRequestToken.RequestID, model.IOSBackgroundCampusTokenRequest)
 
-	res, err := s.Repository.SendBackgroundNotification(notification)
-
-	if err != nil {
+	if _, err := s.Repository.SendBackgroundNotification(notification); err != nil {
 		log.WithError(err).Error("Could not send background notification")
 		return ErrCouldNotSendNotification
 	}
-
-	influx.LogIOSBackgroundRequest(deviceID, campusRequestToken.RequestType, res.Reason)
```

Comment:
@Antonwy What do you need as a metric here exactly? (could you share the dashboards you build?)